component/site/views/flat/icals/tmpl/ical_form.php

```diff
@@ -37,7 +37,7 @@
 	$cats = implode(",", $cats);
 
 	$link = JURI::root() . "index.php?option=com_jevents&task=icals.export&format=ical";
-	if (count($cats) > 0)
+	if (!empty($cats) && count($cats) > 0)
```

Comment:
see above - this is testing the same thing twice

Reply:
set to is_Array